installer/configurators/bitbucket.py

```diff
@@ -0,0 +1,68 @@
+import sys
+import os
+import stat
+import subprocess
+from .common import get_installer_root, get_tfvars_file, replace_tfvars
+
+
+def get_atlassian_tools_path():
+    return get_installer_root() + "/jazz_tmp/atlassian-cli-6.7.1/"
+
+
+def check_bitbucket_user(url, username, passwd):
+    """
+        Check if the bitbucket user is present on the Bitbucket server
+    """
+    bitbucket_sh = get_installer_root() + "/jazz_tmp/atlassian-cli-6.7.1/bitbucket.sh"
+    bitbucket_url = 'http://' + url + ''
+
+    # Add execute bit for the current user to this shell script
+    # since it was downloaded and not checked into Git
+    perms = os.stat(bitbucket_sh)
+    os.chmod(bitbucket_sh, perms.st_mode | stat.S_IEXEC)
+
+    cmd = [
+        bitbucket_sh, '--action', 'createproject', '--project', 'test000',
+        '--name', 'test000', '--server', bitbucket_url, '--user', username,
+        '--password', passwd
+    ]
+
+    try:
+        output = subprocess.check_output(cmd)
```

Comment:
We might need "output = str(subprocess.check_output(cmd))", since it is checked against bytes.

Reply:
Good call, added.